### PR TITLE
Updates for Near Org

### PR DESCRIPTION
**Pull request by calebjacob**
Related To: https://github.com/near/nearorg/issues/68

- Fixed lodash import to support proper tree shaking to resolve unintended package bloat. This reduces our package size by roughly 500KB!
- Added rollup plugin to visualize package output size based on source code and dependencies.
- Adding and modifying various components to support Near Org refactor.

### CODE REVIEW

**Comment by calebjacob** [comment]
Hey @shelegdmitriy I'm going to go ahead and merge this to get a head start on the `nearorg` PR and get these changes published. Let me know if you see any issues. Thanks!

File: src/components/IconCircle.module.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.iconCircle {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 64px;
+  height: 64px;
```
Comment (shelegdmitriy) [inline src/components/IconCircle.module.scss:6]: It would be nice to control icon size, but we can improve this later

**Review by shelegdmitriy — COMMENTED** [review]
LGTM!

File: src/components/IconCircle.module.scss (hunk 1 above)
Comment (calebjacob) [reply src/components/IconCircle.module.scss:6]: Great point! A lot of these components can continue to improve. Feel free to open up tickets and push PR's for any improvements you'd like to see :)

**Pull request merged**